raven.mk

```diff
@@ -9,13 +9,22 @@ SHELL := /bin/bash
 include $(RAVEN_DIR)/amsc.mk
 ###############################################################################
 
+<<<<<<< HEAD
```

Comment:
Conflict

Reply:
There were multiple conflicts in that file and apparently I forgot to check for remaining ones after fixing some.